backend/typescript/models/scheduling.model.ts

```diff
@@ -0,0 +1,38 @@
+import { Column, DataType, Model, Table } from "sequelize-typescript";
+import { Status } from "../types";
+
+@Table({ tableName: "scheduling" })
+export default class Scheduling extends Model {
+  @Column({ type: DataType.INTEGER })
+  id!: number;
+
+  @Column({ type: DataType.INTEGER })
+  donor_id!: string;
+
+  @Column({ type: DataType.STRING })
+  auth_id!: string;
```

Comment:
```suggestion

  @Column({ type: DataType.STRING })
  @Unique 
  auth_id!: string;
```

Reply:
same with this?

Reply:
I know auth_id was included in the notion doc, but I do not think we need that in this table since firebase auth id is being stored in users table. I forgot to update the notion doc.